src/services/positionService.ts

```diff
@@ -713,6 +734,77 @@ export default class PositionService {
     );
   }
 
+  async buildXcallDepositParams(
+    fromToken: Token,
+    toToken: Token,
+    fromValue: string,
+    frequencyType: BigNumber,
+    frequencyValue: string,
+    yieldFromPossible?: string,
+    yieldToPossible?: string,
+    destinantionChainID?: number
+  ) {
+    const destinantionUSDC = this.connextService.getNativeUSDCAddress(destinantionChainID as number);
+
+    const { forwardCallData } = await this.buildDepositParams(
+      fromToken,
+      toToken,
+      fromValue,
+      frequencyType,
+      frequencyValue,
+      yieldFromPossible,
+      yieldToPossible,
+      destinantionUSDC
+    );
+
+    const xTargetAddress = '0xC825d25123Ca8d49066cf8F0AeE164660056e172'; // polygon address for mean target
+
+    const chainID = getChainIdFromUrl();
+    const POLYGON_WETH = '0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619'; // can have any address in UI for destination
+    const destinantionRPC = this.connextService.getRPCURL(destinantionChainID as number);
```

Comment:
already a number

Reply:
It's an optional property so could be undefined sometimes, so TS throws error, Property is `number|undefined`